packages/binding-http/src/http-server.ts

```diff
@@ -487,7 +487,11 @@ export default class HttpServer implements ProtocolServer {
         }
     }
 
-    private parseUrlParameters(url: string, uriVariables: { [key: string]: TD.DataSchema }): Record<string, unknown> {
+    private parseUrlParameters(
```

Comment:
DRY: this is a duplication of coap-server#parseUrlParameter ?  may be a shared utility method is needed instead.